array-destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,13 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPrice: 1.0 },
   { itemName: "Hash Brown", quantity: 4, unitPrice: 0.4 },
 ];
+
+console.log(`QTY\tITEM\t\t\tTOTAL`);
+let total = 0;
+for (const item of order) {
+  let {itemName, quantity, unitPrice} = item;
+  console.log(`${quantity}\t${itemName.padEnd(18)}\t${(unitPrice * quantity).toFixed(2)}`);
+  total += unitPrice * quantity
+}
+console.log();
+console.log(`Total: ${total}`);
```

Comment:
Well done for finding a valid solution, a few potential improvements:

Is this a use case for the `reduce` higher-order function? Refer to this [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce).

Is the `console.log();` on line 17 required? Is there a way to achieve the same result just by using the log on line 18?

Reply:
Also, `.padEnd(18)` would seem like an arbitrary amount of whitespace to add to this string, would be easier to read for others if you were consistent with how you added whitespace on the logs (refer to how you did it on line 10)